companion/lib/Surface/USB/Loupedeck.ts

```diff
@@ -312,8 +356,34 @@ export class SurfaceUSBLoupedeck extends EventEmitter<SurfacePanelEvents> implem
 					})
 			} else if (control.feedbackType === 'lcd') {
 				this.#writeQueue.queue(control.id, render)
+			} 
+		} else if (control.type === 'lcd-segment') { // Update the slider display on render call to change color
+				const styleBg =
+					typeof render.style === 'object' && render.style !== null && 'bgcolor' in render.style
+						? (render.style as any).bgcolor
+						: undefined
```

Comment:
This can be simplified and kept type safe:
```suggestion
				const styleBg = render.style?.bgcolor
```